.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,25 @@
 <!--
-Note: This checklist is a reminder of our shared engineering expectations. Feel free to change it, although assigning a GitHub reviewer and the items in bold are required.
+Note: This checklist is a reminder of our shared engineering expectations. 
+The items in Bold are required
+If your PR involves UI changes:
+    1. Upload screenshots or screencasts that illustrate the changes before / after
+    2. Add them under the UI changes section (feel free to add more columns if needed)
+    3. Make sure these changes are tested in API 23 and API 26
```

Comment:
why 23 and not 21?

Reply:
good catch @CDRussell , that should be a 21!